### ISSUE TITLE
Using jQuery listener on events fired from Prototype on document.

### ISSUE THREAD

**Issue by braincomb** [Widen/jquery-prototype-custom-event-bridge#4]
Hi!

Thanks for this script! it works great when used on elements, however if using `document.fire("custom:event")` then I am unable to catch this event with jQuery:

```javascript
jQuery(document).on("custom:event", function() {
  console.log("got it");
});

document.fire("custom:event");
```
The above will not work. I also noticed that the test for this case is missing, is this on purpose because it's something that's not possible?

Thanks for your help.

**Comment by rnicholus** [comment]
You're right, this is a gap in the bridge, an unintended one.

Prototype has special code for the `fire` function that is adds to the `Document` interface. I think the problem may be solved simply by adding the following to the end of the bridge code:

```javascript
Object.extend(document, {
   fire: function(eventName, memo) {
      if (eventName.indexOf(':') > 0) {
         oldjQueryTrigger(eventName, memo ? [memo] : null, document);
      }
      oldPrototypeFire(document, eventName, memo);
    }
});
```

If this works and all tests are passing, I'll look to update the library.

**Comment by rnicholus** [comment]
Fixed in 1.0.2, just pushed to npm as well.
